### PR TITLE
windows pipes

### PR DESCRIPTION
**Pull request by Jarred-Sumner**
### What does this PR do?

i want to see the test progress bar

- [ ] Documentation or TypeScript types (it's okay to leave the rest blank in this case)
- [ ] Code changes

### CODE REVIEW

File: src/bun.js/api/bun/subprocess.zig [hunk 1]
```diff
@@ -886,26 +965,54 @@ pub const Subprocess = struct {
...
+                // fixed capacity
+                .internal_buffer = bun.ByteList.initWithBuffer(slice),
+                .auto_sizer = null,
+                .stream = pipe,
+            };
+        }
+
         pub fn initWithAllocator(allocator: std.mem.Allocator, fd: bun.FileDescriptor, max_size: u32) BufferedOutput {
+            if (Environment.isWindows) {
+                @compileError("Cannot use BufferedOutput with fd on Windows use please use .initWithPipeAndAllocator");
```
Comment (jdalton) [inline src/bun.js/api/bun/subprocess.zig]:
```suggestion
                @compileError("Cannot use BufferedOutput with fd on Windows please use .initWithPipeAndAllocator");
```

File: src/sys_uv.zig [hunk 2]
```diff
@@ -350,13 +350,13 @@ pub fn pwritev(fd: FileDescriptor, bufs: []const bun.PlatformIOVec, position: i6
...
         }
-        log("uv write({}, {d} total bytes) = {d} ({any})", .{ uv_fd, total_bytes, rc.value, debug_timer });
+        log("uv write({}, {d} total bytes) = {d} ({any})", .{ uv_fd, total_bytes, rc.int(), debug_timer });
     }
 
     if (rc.errno()) |errno| {
         return .{ .err = .{ .errno = errno, .fd = fd, .syscall = .write, .from_libuv = true } };
     } else {
-        return .{ .result = @as(usize, @intCast(rc.value)) };
+        return .{ .result = @as(usize, @intCast(rc.int())) };
```
Comment (jdalton) [inline src/sys_uv.zig:359]: What's the `.value` to `.int()` change about?

Comment (cirospaciari) [reply src/sys_uv.zig:359]: `ReturnCode` was a struct and now is a enum, so we dont have a `.value` anymore

**Comment by Jarred-Sumner** [comment]
Great. Now just need to figure out why it takes 2 hours to run the tests

**Pull request merged**